src/scripts.js

```diff
@@ -11,28 +11,65 @@ import User from './user';
 import Cookbook from './cookbook';
 
 let favButton = document.querySelector('.view-favorites');
-let homeButton = document.querySelector('.home')
+let recipeToCookButton = document.querySelector('.view-recipe-to-cook');
+let homeButton = document.querySelector('.home');
+let searchButton = document.querySelector('#search-button');
 let cardArea = document.querySelector('.all-cards');
 let cookbook = new Cookbook(recipeData);
 let user, pantry;
 
 window.onload = onStartup();
 
 homeButton.addEventListener('click', cardButtonConditionals);
+recipeToCookButton.addEventListener('click', viewRecipeToCook);
 favButton.addEventListener('click', viewFavorites);
 cardArea.addEventListener('click', cardButtonConditionals);
+searchButton.addEventListener('click', searchRecipe);
 
+function searchRecipe() {
```

Comment:
For the search functionality, we should consider using toLowerCase() on the search input and the titles that it searches through to account for user error.  If the user doesn't have the matching capitalization, it might not appear for them even if they have the correct name and spelling.

Reply:
You are right!  I will make the adjustments so that the search feature is working properly when user input words with upper or lower cases.